### PR TITLE
Command on Startup

### PR DESCRIPTION
**Pull request by Atomfusion1**
Command on Startup per Character loading /lua or /macro or even /hidecorpse looted per character 
added padding for /nav .. You do not need to be as close as afollow

### CODE REVIEW

File: E3Next/Processors/Movement.cs [hunk 1]
```diff
@@ -74,7 +74,7 @@ public static void Check_Chase()
                                     bool pathExists = MQ.Query<bool>($"${{Navigation.PathExists[id {spawnID}]}}");
                                     if (pathExists)
                                     {
-                                        MQ.Cmd($"/squelch /nav id {spawnID} log=error");
+                                        MQ.Cmd($"/squelch /nav id {spawnID} dist=20 log=error");
```
Comment (RekkasGit) [inline E3Next/Processors/Movement.cs:77]: this has nothing to do with command on startup. 
also hard coded values off the normal default behavior will need an option in the general settings or character settings.

File: E3Next/E3.cs [hunk 2]
```diff
@@ -264,7 +264,11 @@ private static void Init()
                 //setup is done after the settings are setup.
                 //as there is an order dependecy
                 Setup.Init();
-
+                foreach (var SendCommand in E3.CharacterSettings.Misc_CommandOnStartup)
+                {
+                    MQ.Write(SendCommand);
```
Comment (RekkasGit) [inline E3Next/E3.cs:269]: can probably put some color on this and be a bit more descriptive.

File: E3Next/Processors/Movement.cs (hunk 1 above)
Comment (Atomfusion1) [reply E3Next/Processors/Movement.cs:77]: I Have just removed this change atm as I agree it should be adjustable .. but just a note It currently is hard coded .. and all my people just ping pong around with selos in group as everyone tries to be on top

File: E3Next/E3.cs (hunk 2 above)
Comment (Atomfusion1) [reply E3Next/E3.cs]: Updated

**Comment by RekkasGit** [comment]
After discussion with the team, ultimately going to have to forgo this feature for now. 

Work arounds are currently available as
1)  ingame.cfg
2)  privatecode.cs

Having out of bounds private code is an issue that will be more headache than its worth. 
Where the event loop is in the process of doing one thing then the other code decides to do another. 

Now, I believe it is possible to do compile time of things at startup, off a text based *.cs file
I will look into this in the future, but its going to be a few months

**Pull request closed without merging**